interface/portraits/portraits_tauren.gfx

```diff
@@ -13,7 +13,17 @@ spriteTypes = {
 			"GFX_character_background:p0"
```

Comment:
Do you see these black lines?
Is it the result of low-quality cropping?
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/105497376-306be200-5cd8-11eb-9ae7-dbe7dd1f74f3.png)
![image](https://user-images.githubusercontent.com/46384992/105497390-35c92c80-5cd8-11eb-924b-6d4e9ea9b099.png)

</details>

Reply:
These black lines again.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/105498063-20083700-5cd9-11eb-8150-d7443b24def8.png)

</details>

Reply:
> These black lines again.

This one also has sharp edges because looks like you delete the alpha channel in the process.